tests/admin_utils/models.py

```diff
@@ -5,6 +5,7 @@
 
 class Site(models.Model):
     domain = models.CharField(max_length=100)
+    owner = models.CharField(verbose_name=_("Site owner"))
```

Comment:
```suggestion
    owner = models.CharField(verbose_name="Site owner", blank=True, null=True)
```
(then we can remove the changes in `tests/admin_utils/test_logentry.py`)